Extensions/addressbar.js

```diff
@@ -0,0 +1,81 @@
+//* TITLE Addressbar Updater **//
+//* VERSION 1.0.1 **//
+//* DESCRIPTION Keeps the url in its place **//
+//* DEVELOPER srsutherland **//
+//* DETAILS Sets the url to the current post on your dashboard (to keep your place during tab crashes and refreshes). **//
+//* FRAME false **//
+//* BETA false **//
+//* SLOW false **//
+
+XKit.extensions.addressbar = new Object({
+
+	running: false,
+	interval: 0,
+	initialScroll: null,
+	
+	preferences: {
+		"focus_only": {
+			text: "Only run the extension when the page has focus",
+			default: true,
+			value: true
+		},
+	},
+
+	run: function() {
+		this.running = true; //there's some confusion over whether this gets set to true on immediate return
+		
+		let XInterface = XKit.interface.where()
```

Comment:
since XKit extensions are run per pageload it might be clearer to use `const` rather than `let` here